src/sass/_shame.scss

```diff
@@ -5,11 +5,12 @@
 
 // message while loading react app
 .loading-message {
-  text-align: center;
   padding: 2em 0;
+  text-align: center;
 }
 
-body .row {
+
+.row {
```

Comment:
All of the changes are of the following type:
-alphabetization of properties
-removing leading zeros from decimal values
-removing units from zero values
-trailing whitespace/missing padding
-double colons for pseudo-elements
-using // instead of /**/ for single line comments
-wrapping strings in double quotes
-removing qualifying elements for class selectors for `body .row` and `a.usa-button-primary` within both `.expander-content-inner` and `.expander-content-closed`, as
--it is unlikely that classes will be assigned outside of the `body` element
--`usa-button-primary` is only ever used for anchor tags within `.expander-content-inner` and `.expander-content-closed`
As a result, these will not affect the styling behavior. These were all recommended by the linter along with some other recommendations (for example, using `!important` and id's were also discouraged, but fixing these seemed likely to necessitate additional changes elsewhere). Please let me know if you prefer to revert any of these.